doc/decisions/0015-current-executed-file-in-runner.md

```diff
@@ -0,0 +1,20 @@
+# 15. Current executed file in Test  Runner
```

Comment:
When I suggested adding an ADR I was referring not to this feature in particular, but to the strong assumption that the tests will be loaded and run sequentially.

So the title could be something like "Sequencial loading and execution of suites and tests"

We can refer to this feature as an example.

Reply:
Oh sorry I misunderstood you!